js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.html

```diff
@@ -82,6 +82,35 @@ <h3 data-bind="css: $component.classes('analysis-name'), text: ($data.domainId ?
                             </div>
                         <!-- /ko -->
                     </div>
+
+                    <!-- ko foreach: $data.stratas -->
```

Comment:
Why stratas go into a different table?

Reply:
I believe we don't want to mix stratified results with ordinal analyses results, am I wrong or misunderstood your question?